_posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown

```diff
@@ -0,0 +1,401 @@
+---
+layout: post
+title:  "How to use Fluent Bit and OpenSearch with Bottlerocket and Kubelet logs"
```

Comment:
Could we shorten this to 'Using Fluent Bit...', just seems like a really long title.

Reply:
I like it.